AcademyApp/ViewControllers/LoginViewController.swift

```diff
@@ -80,151 +85,131 @@ extension LoginViewController : UITextFieldDelegate
+
+    func setupUI() {
         backgroundColorSetup()
         textFieldsSetup()
-        showLabelSetup()
+        showButtonSetup()
         horizontalLinesSetup()
         doneButtonSetup()
-        
-        defaultDoneButtonLoginFormVerticalSpacingConstant = doneButtonLoginFormVerticalSpacingConstraint.constant
-        
+
+        defaultVerticalSpacingConstant = verticalSpacingConstraint.constant
     }
-    
-    func backgroundColorSetup(){
+
+    func backgroundColorSetup() {
         view.backgroundColor = UIColor.almostBlack
     }
-    
-    func horizontalLinesSetup(){
+
+    func horizontalLinesSetup() {
         horizontalLineViews.forEach {
             $0.backgroundColor = UIColor.blackish
         }
     }
-    
-    func textFieldsSetup(){
-        
+
+    func textFieldsSetup() {
+
         textFields.forEach {
             $0.attributedPlaceholder = NSAttributedString(string: $0.placeholder ?? "", attributes: [
-                NSAttributedString.Key.foregroundColor : UIColor.brownGray,
-                NSAttributedString.Key.font : UIFont.showLabelFont,
-                NSAttributedString.Key.kern : 1
+                NSAttributedString.Key.foregroundColor: UIColor.brownGray,
+                NSAttributedString.Key.font: UIFont.showLabelFont,
+                NSAttributedString.Key.kern: 1
                 ])
-            
+
             $0.textColor = UIColor.white
             $0.keyboardAppearance = .dark
         }
     }
-    
-    func showLabelSetup(){
+
+    func showButtonSetup() {
         showButton.isHidden = true
-        
+
         let attributedTitle = NSAttributedString(string: showButton.title(for: .normal) ?? "SHOW", attributes: [
-            NSAttributedString.Key.foregroundColor : UIColor.brownGray,
-            NSAttributedString.Key.font : UIFont.showLabelFont,
-            NSAttributedString.Key.kern : 1
+            NSAttributedString.Key.foregroundColor: UIColor.brownGray,
+            NSAttributedString.Key.font: UIFont.showLabelFont,
+            NSAttributedString.Key.kern: 1
             ])
-        
+
         showButton.setAttributedTitle(attributedTitle, for: .normal)
     }
-    
-    func doneButtonSetup(){
-        doneButton.isEnabled = false
-        
-        let attributedTitle = NSAttributedString(string: doneButton.title(for: .normal) ?? "Done", attributes: [
-            NSAttributedString.Key.font : UIFont.doneButtonTitleFont,
-            NSAttributedString.Key.foregroundColor : UIColor.white])
-        
-        doneButton.setAttributedTitle(attributedTitle, for: .normal)
-    }
-}
 
-// MARK: - Constants
+    func doneButtonSetup() {
+        doneButton.isEnabled = false
 
-private extension LoginViewController
-{
-    struct Config {
-        static let adjustedDoneButtonLoginFormVerticalSpacing : CGFloat = 56
+        doneButton.setTitle(doneButton.titleLabel?.text ?? "Done", for: .normal)
+        doneButton.titleLabel?.font = UIFont.doneButtonTitleFont
```

Comment:
UIFOnt & UIColor not needed here... if its obvious its not necessary to write class for enum etc

Reply:
Remove unnecessary class names.